### PR TITLE
Feature: add Helm repository

### PR DESCRIPTION
**Pull request by czerasz-flaconi**
Make it possible to add Helm repositories.

Example:

```yaml
helm_repositories:
  - name: zalenium-github
    url: https://raw.githubusercontent.com/zalando/zalenium/3.141.59u/charts/zalenium
```

### CODE REVIEW

File: library/parse_helm_repositories.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#!/bin/env python
```
Comment (cytopia) [inline library/parse_helm_repositories.py]: This should be `/usr/bin/env`

Comment (cytopia) [inline library/parse_helm_repositories.py]: Additionally we should be using Python3 or? `#!/usr/bin/env python3`

**Review by cytopia — COMMENTED** [review]
Just found during a quick check

**Review by zxpower — DISMISSED** [review]
👍

File: library/parse_helm_repositories.py (hunk 1 above)
Comment (czerasz-flaconi) [reply library/parse_helm_repositories.py]: The unit tests tun on what is in the container.

But the module is created to work with both versions:

```bash
docker run --rm -it \
  -v ${PWD}/:/tests/ \
  -w /tests \
  --entrypoint /bin/sh \
  python:3 -c "pip3 install ansible==${ANSIBLE_VERSION} && python3 /tests/library/test_parse_helm_repositories.py"
Collecting ansible==2.5
  Downloading https://files.pythonhosted.org/packages/4a/3b/9d98e132074bb6a3f18fd811db2819fbde6fc8a26fad9a40b49e53cb2455/ansible-2.5.0.tar.gz (10.1MB)
...
Building wheels for collected packages: ansible, PyYAML, pycparser
  Building wheel for ansible (setup.py) ... done
  Stored in directory: /root/.cache/pip/wheels/9f/48/10/6c5633adf264de3c5b49a66397a01cd5ee9653fa6155b34cf4
  Building wheel for PyYAML (setup.py) ... done
  Stored in directory: /root/.cache/pip/wheels/d9/45/dd/65f0b38450c47cf7e5312883deb97d065e030c5cca0a365030
  Building wheel for pycparser (setup.py) ... done
  Stored in directory: /root/.cache/pip/wheels/f2/9a/90/de94f8556265ddc9d9c8b271b0f63e57b26fb1d67a45564511
Successfully built ansible PyYAML pycparser
Installing collected packages: MarkupSafe, jinja2, PyYAML, pycparser, cffi, six, cryptography, pynacl, bcrypt, paramiko, ansible
Successfully installed MarkupSafe-1.1.1 PyYAML-5.1.2 ansible-2.5.0 bcrypt-3.1.7 cffi-1.13.1 cryptography-2.8 jinja2-2.10.3 paramiko-2.6.0 pycparser-2.19 pynacl-1.3.0 six-1.12.0
You are using pip version 19.0.1, however version 19.3.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
..
----------------------------------------------------------------------
Ran 2 tests in 0.000s

OK
```

@cytopia shall I add to `test-unit` target the following?

```bash
docker run --rm -it \
  -v ${PWD}/:/tests/ \
  -w /tests \
  --entrypoint /bin/sh \
  python:3 -c "pip3 install ansible==${ANSIBLE_VERSION} && python3 /tests/library/test_parse_helm_repositories.py"
```

This way we test both `python` versions. Though this slows down CI.

Comment (czerasz-flaconi) [reply library/parse_helm_repositories.py]: Good catch. Fixed!

Comment (cytopia) [reply library/parse_helm_repositories.py]: > This way we test both python versions. Though this slows down CI.

Having both specifically is good I think. Jobs can run in parallel or?

**Review by zxpower — DISMISSED** [review]
And again 👍

File: library/parse_helm_repositories.py (hunk 1 above)
Comment (czerasz-flaconi) [reply library/parse_helm_repositories.py]: Testing now with Python 3.

**Review by zxpower — DISMISSED** [review]
💯

**Review by zxpower — APPROVED** [review]
LGTM!

**Pull request merged**